src/Employee/Report.java

```diff
@@ -0,0 +1,15 @@
+package Employee;
+
+public class Report {
+    public static void main(String[] args) {
+        Employee client = new Employee();
+        Employee bankAc = new Employee();
+        Employee methods = new Employee();
```

Comment:
'methods' name object is not very relevant because a real object of the Employee class is some employee as a person